client/src/Components/Create/Create.js

```diff
@@ -1,12 +1,122 @@
 import React, { Component } from 'react';
 import './Create.css';
+import Schedule from '../Schedule/Schedule';
+import Location from '../Location/Location';
+import jQuery from 'jquery';
+var $ = window.jQuery;
+
+$ = $ || jQuery;
+
+$.fn.locationpicker = $.fn.locationpicker || function() {};
 
 class Create extends Component {
+  componentDidMount() {
+    $('#location-field').locationpicker({
+      radius: 300,
+      location: {
+        latitude: 47.6062,
+        longitude: -122.3321
+      },
+      inputBinding: {
+        radiusInput: $('[name=radius]')
+      }
+    });
+  }
+
   render() {
     return (
       <div>
-        <h1>Create</h1>
-        <div>This is a test.</div>
+        <div className="container">
+          <div className="row">
+            {/*creation of new meeting title*/}
+            <div className="col-sm-3"></div>
+            <div className="col-sm-6">
+              <h1 id="title">Create a Meeting</h1>
+            </div>
+            <div className="col-sm-3"></div>
+          </div>
+
+          {/*creation of new meeting form*/}
+          <div className="row" id="response-row">
+            <div className="col-sm-3"></div>
+            <div className="col-sm-6">
+              <form id="creation-form" onSubmit={this.submit}>
+                {/*title field*/}
+                <div className="form-group">
+                  <label>Meeting Name</label>
+                  <input name="title" className="form-control" type="text" placeholder="Type the meeting's name" required />
+                </div>
+                {/*closeout time field*/}
+                <div className="form-group">
+                  <label>Close-out Time</label>
+                  <input name="closeoutTime" className="form-control" type="text" placeholder="Type the time the meeting should lock in" required />
```

Comment:
Do we need some kind of input validation or some kind of guideline for how this input should look?